base_layer/core/src/transactions/transaction_protocol/transaction_initializer.rs

```diff
@@ -158,6 +160,12 @@ impl SenderTransactionInitializer {
         self
     }
 
+    /// Enable or disable spending of dust less than the fee
+    pub fn with_dust_collector(&mut self, dust_collector_enabled: bool) -> &mut Self {
```

Comment:
You don't need an `Option` here. There is going to be a default, so set the default in the builder constructor and this function can override it if it's called.